webstruct/model.py

```diff
@@ -136,15 +139,17 @@ def _join_tokens(html_tokens):
     return smart_join(t.token for t in html_tokens)
 
 
-def extract_entitiy_groups(html_tokens, tags, dont_penalize=None,
-                           join_tokens=_join_tokens):
+def extract_entitiy_groups(html_tokens, tags, seq_enc, bilou=False,
```

Comment:
let's use more verbose sequence_encoder name, at least to be consistent with HtmlTokenizer